bump.sh

```diff
@@ -21,3 +21,13 @@ echo "Starting ${version} version bump"
 
 # Generate CHANGELOG and create a commit (see .versionrc.json)
 npx standard-version --release-as ${version}
+
+# I am sorry.
+#
+# I've gone diving through the code of `conventional-changelog` to see if there
+# was a way to configure the string and ultimately I decided that a 'sed' was the simpler
+# way to go.
+sed -i.tmp -e 's/BREAKING CHANGES$/BREAKING CHANGES TO EXPERIMENTAL FEATURES/' CHANGELOG.md
+rm CHANGELOG.md.tmp
+git add CHANGELOG.md
+git commit --amend --no-edit
```

Comment:
was the alternative you considered before landing on `sed`? 
making use of the lifecycle scripts? or other?